qiskit/visualization/matplotlib.py

```diff
@@ -347,9 +347,16 @@ def _get_colors(self, op):
             ec = self._style.edge_color
             lc = self._style.lc
         if op.name == 'reset':
+            gt = self._style.lc
+        else:
+            gt = self._style.gt
+        if op.name not in {'h', 't', 's', 'z', 'sdg', 'tdg', 'u1', 'r', 'reset', 'meas', 'measure'}:
```

Comment:
```suggestion
        if op.name not in {'h', 't', 's', 'z', 'sdg', 'tdg', 'u1', 'reset', 'meas', 'measure'}:
```

I think `r` should be removed because it seems like its a quantum gate which needs a white font.

Binder Tests
![image](https://user-images.githubusercontent.com/29938963/93553050-eb4d3300-f926-11ea-9b52-20a21e451640.png)